mmv1/templates/terraform/examples/cloud_run_service_secret_environment_variables.tf.erb

```diff
@@ -0,0 +1,81 @@
+data "google_project" "project" {
+  provider = google-beta
+}
+
+resource "google_project_service" "secret-manager" {
```

Comment:
This test runs in the default CI project, so this API should be enabled by default. Lets remove this, as when the test finishes this resource is destroyed which disables the API, causing other in-process tests to fail.

Reply:
Removed from this test and the volumes one.